Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/ContentDialogSamplePage.xaml.cs

```diff
@@ -31,8 +31,16 @@ private async void Button_Click(object sender, RoutedEventArgs e)
 			{
 				Content = "Hello world!",
 				PrimaryButtonText = "OK",
-				XamlRoot = this.XamlRoot
+				XamlRoot = this.XamlRoot,
 			};
+
+			if ((sender is Button button) && button.Tag is { } tag && tag.ToString() == "Fluent")
```

Comment:
I feel this is shorter and easy to read. How do you feel about this change?
```suggestion
			if ((sender as Button)?.Tag?.ToString()  == "Fluent")
```

Reply:
Nice changes! Definitely shorter and easier, thanks, Andres!